sql/src/main/java/io/crate/executor/transport/executionphases/InitializationTracker.java

```diff
@@ -26,6 +26,19 @@
 
 import java.util.concurrent.atomic.AtomicInteger;
 
+/**
+ * Triggers {@link #future} after {@code numServer} calls to either
+ * {@link #jobInitialized()} or {@link #jobInitializationFailed(Throwable)}
+ * <p>
+ *     If {@link #jobInitializationFailed(Throwable)}} has been called at least once
+ *     the future will emit a failure.
+ * </p>
+ * <p>
+ *     The failure emitted will be the last failure received via {@link #jobInitializationFailed(Throwable)}.
+ *     The exception being InterruptedException:
+ *     It has a lower priority and won't overwrite previous failures.
```

Comment:
The code is just written the other way around: only overwrite if the previous failure is null or InterruptedException. 
Should I change the comment?